vignettes/data-transform-as-shiny-module.Rmd

```diff
@@ -12,13 +12,13 @@ vignette: >
 
 ## Introduction
 
-`teal` version `0.16` introduced new argument in `teal::module` called `transformers`. 
+`teal` version `0.16` introduced new argument in `teal::module` called `transformators`. 
 This argument allows to pass a `list` of `teal_data_module` objects that are created using `teal_transform_module()` function.
```

Comment:
```suggestion
This argument allows to pass a `list` of `"teal_data_module"` class of objects created using [`teal_transform_module()`](https://insightsengineering.github.io/teal/main/reference/teal_transform_module.html) function.
```

Reply:
- put class name as a string
- if you refer to function - link it - it would be easier to navigate through the docs